navapbc/labs-decision-support-tool
# feat: add similarity score to accordion value

## Ticket

https://navalabs.atlassian.net/browse/DST-333


## Changes
> added `response_format` param for determining if response will return a tuple with content and score or just content
> added metadata field in onMessage call to send information to literal ai


## Testing

1.  Navigate to `localhost:8000/chat` and enter query
2.  Expand accordion, similarity score value should appear in body of text

<img width="703" alt="Screenshot 2024-07-17 at 5 17 27 PM" src="https://github.com/user-attachments/assets/611dcce8-af8c-4272-bcfe-84036a2b8504">

1. Navigate to LiteralAI page 
2. click on Threads
3. click into thread query and scroll to metadata, title with similarity score should appear in metadata
<img width="1252" alt="Screenshot 2024-07-17 at 5 19 39 PM" src="https://github.com/user-attachments/assets/cf9df32d-8369-4ea3-b2dc-0aab9a8be815">
